draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -1131,6 +1131,11 @@ explicitly. Horizontal tab (%x09) is excluded from the CTL characters that
 lead to set-cookie-string rejection, as it is considered whitespace, which is
 handled separately.
 
+NOTE: The set-cookie-string may contain octet sequences that appear
+percent-encoded as per {{Section 2.1 of RFC3986}}. However, a user agent
```

Comment:
I think the section number goes outside of the {{}}.

Reply:
With `{{Section 2.1 of RFC3986}}` it creates an additional deeplink into that section:
![image](https://user-images.githubusercontent.com/2622601/151442613-e4f1c7d2-9858-43f2-a970-8128f6459b69.png)

Which I think is nice, so I'll keep it like that if you don't mind :)

Reply:
TIL, that's great. :)